Madness/Map.swift

```diff
@@ -24,3 +42,11 @@ infix operator >>- {
 	associativity left
 	precedence 150
 }
+
+/// Map operator.
+infix operator <^> {
+	associativity left
+
+	// Higher precedence than `>>-`.
+	precedence 155
```

Comment:
Yes, I did check Runes, and felt a little conflicted here. I should create an issue over there, because I think it would be helpful to standardise where possible.